backends/bmv2/common/expression.cpp

```diff
@@ -403,6 +429,15 @@ void ExpressionConverter::postorder(const IR::Member* expression)  {
                 } else if (lv->is<Util::JsonValue>()) {
                     e->append(lv);
                     e->append(fieldName);
+                } else if (auto jo = l->to<Util::JsonObject>()) {
+                    if (st) {
```

Comment:
what happens if st is false? If it cannot be false then please add a `BUG_CHECK(st)`.

Reply:
If the P4 program does not use runtime index, then `st` is false and this else if will not even be hit:

https://github.com/hesingh/p4c/blob/master/backends/bmv2/common/expression.cpp#L432